src/libraries/Microsoft.PowerFx.Core/Utils/CharacterUtils.cs

```diff
@@ -380,6 +384,18 @@ private static void UpdateEscapeInternals(string escapedValue, string input, int
             sb.Append(escapedValue);
         }
 
+        private static void HandleEscapedCurlyBraces(char curlyBrace, string input, bool isInterpolatedInput, int estimatedLength, int currentPosition, ref int charsToAdd, ref StringBuilder sb)
+        {
+            Contracts.AssertOneOfValueType(curlyBrace, '{', '}');
+
+            if (!isInterpolatedInput)
+            {
+                return;
```

Comment:
This function does not need to accept this boolean parameter just to return if it's false. It would be better to have this if statement in the switch/case body.

Reply:
Done. Removed this function and moved the code directly in the case block.